duolingo_sync/plugin.py

```diff
@@ -64,14 +64,12 @@ def login_and_retrieve_vocab(username, password) -> VocabRetrieveResult:
                 """
                 <p>Logging in to Duolingo failed. Please check your Duolingo credentials.</p>
 
-                <p>Having trouble logging in? You must use your <i>Duolingo</i> username and password.
```

Comment:
Edited this copy in https://github.com/JASchilz/AnkiSyncDuolingo/pull/65/files, which I based this PR on. Not sure this copy really makes sense in the context of embedded authentication.